pubpub/pubpub
# update: Terms of Service

Updates the TOS, and fixes some longstanding CSS weirdness that was causing abnormally large text sizes for list items.

_Test Plan_
- Visit `/tos`
- Read and make sure it matches the attached uploaded docx.
- Check all the links – 1 to CC and 4 email links – and make sure they work as expected.

[PubPub Terms of Service - 31 Oct 19 - CLEAN FINAL (1).docx](https://github.com/pubpub/pubpub/files/4119945/PubPub.Terms.of.Service.-.31.Oct.19.-.CLEAN.FINAL.1.docx)
